MergePythonSDK/accounting/apis/__init__.py

```diff
@@ -0,0 +1,49 @@
+
+# flake8: noqa
+
+# Import all APIs into this package.
+# If you have many APIs here with many many models used in each API this may
+# raise a `RecursionError`.
+# In order to avoid this, import only the API that you directly need like:
+#
+#   from MergePythonSDK.accounting.api.account_details_api import AccountDetailsApi
+#
+# or import this package, but before doing it, use:
+#
+#   import sys
+#   sys.setrecursionlimit(n)
+
+# Import APIs into API package:
+from MergePythonSDK.accounting.api.account_details_api import AccountDetailsApi
```

Comment:
I noticed this differs from the `__init__.py` module declaration of the `/api` (singular) folder, is this a relic from what the openapi generator spits out by default? If so please delete this entire folder. same with `/models` vs `/model`